forms-flow-api/src/formsflow_api/services/import_support.py

```diff
@@ -726,8 +726,6 @@ def import_form_workflow(
                 else:
                     raise BusinessException(BusinessErrorCode.INVALID_FILE_TYPE)
             elif valid_file == ".bpmn":
-                if CREATE_BPMN_FLOWS not in user.roles:

```

Comment:
why this removed

Reply:
since now we're not restricting workflow edit with create_bpmn_flows in opensrc  based on create_bpmn_flows permission